### PR TITLE
basic manifest page

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#7260](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7260)
Task [AB#7536](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7536)

image

### CODE REVIEW

**Comment by alibombaye** [comment]
The "Select list price" heading will come from the mainAdvice of the question. As with the selectSolution

File: app/pages/sections/catalogue-solutions/price/contextCreator.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    it('should return the description', () => {
+      const context = getContext({ orderId: 'order-1' });
+      expect(context.description).toEqual(manifest.description);
+    });
+
+    it('should return the listPriceHeading', () => {
+      const context = getContext({ orderId: 'order-1' });
+      expect(context.listPriceHeading).toEqual(manifest.listPriceHeading);
+    });
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/price/contextCreator.test.js:31]: you can remove this

File: app/pages/sections/catalogue-solutions/price/manifest.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "backLinkText": "Go back",
+  "title": "List price for",
+  "description": "Select the list price for this Catalogue Solution. If you’ve agreed a different price with the supplier, the values can be altered later.",
+  "listPriceHeading": "Select list price",
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/price/manifest.json:5]: remove this.

File: app/pages/sections/catalogue-solutions/price/template.njk [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{% extends 'includes/layout.njk' %}
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/price/template.njk:1]: rename the folder to select-price

File: app/pages/sections/catalogue-solutions/price/template.njk [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        text: backLinkText
+      }) }}
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 data-test-id="solutions-price-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
+        <h2 data-test-id="solutions-price-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
+
+        <h3 data-test-id="order-description-heading" class="nhsuk-u-margin-bottom-3">{{ listPriceHeading }}</h3>
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/price/template.njk:20]: remove this

File: app/pages/sections/catalogue-solutions/price/template.test.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import { componentTester } from '../../../../test-utils/componentTester';
+import manifest from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/sections/catalogue-solutions/price/template.njk',
+  },
+};
+
+describe('catalogue-solutions page', () => {
```
Comment (alibombaye) [inline app/pages/sections/catalogue-solutions/price/template.test.js:10]: add `select-price` to description

File: app/pages/sections/catalogue-solutions/price/manifest.json (hunk 2 above)
Comment (kat1906) [reply app/pages/sections/catalogue-solutions/price/manifest.json:5]: why?

Comment (kat1906) [reply app/pages/sections/catalogue-solutions/price/manifest.json:5]: I think it is right as there is a heading of "Select list price"

**Pull request merged**